docs/component-as-prop.md

```diff
@@ -15,3 +15,35 @@ This also has some added advantages:
 
 - The _owner_ is the one controlling how the component renders, not some opaque logic inside `MyBanner`.
 - Circumvents the tendency of needing a [props spread](props-spread.md) in `MyBanner`'s render, since passing a component to a child and the child using prop spread on said component usually go together.
+
+--- 
+
+In practise, if you want a component that would accept children, you can do the following:
```

Comment:
hey ! there's a small typo here "In pratise" > "in practice"

Reply:
`if you want a component that accepts children`. + I'd invert the code samples and put `then you can use it like this:` between the two